contrib/fastd-exporter-start.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Start fastd instances
+fastd_instances=("dom0" "dom1" "dom2")
+
+# Array to hold the PIDs of the fastd instances
+fastd_pids=()
+
+# Stop the fastd instances when the script exits
+trap 'for pid in "${fastd_pids[@]}"; do kill $pid; done' EXIT
+
+# Start the fastd instances
+for instance in "${fastd_instances[@]}"
+do
+    fastd --config $SCRIPT_DIR/fastd/$instance/fastd.conf --status-socket $SCRIPT_DIR/fastd-$instance.sock 2>&1 | sed "s/^/[fastd $instance] /" &
```

Comment:
I don't think the stream output provides any additional value

Reply:
i think it's quite helpful to determine wheter fastd was succesfully started and the connections where established as expected.